File: c/202012-1.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
	int n;
	scanf("%d", &n);

	int score = 0;
	int i, wi, scorei;
	for (i = 0; i < n; i++) 
	{
		scanf("%d %d", &wi, &scorei);
		score += wi * scorei;
	}

	score = score > 0 ? score : 0;
	printf("%d\n", score);
	return 0;
}

File: python/202012-1.py
def main():
	n = int(input())

	score = 0
	for i in range(n):
		arr = input().split(' ')
		w_i, score_i = int(arr[0]), int(arr[1])
		score += w_i * score_i
	score = max(0, score)
	print(score)

if __name__ == '__main__':
	main()